### PR TITLE
add systemjs-plugin-json@0.2.2 w/ git autoupdate

### LINKED ISSUE #9641: [Request] systemjs-plugin-json
**Library name:** systemjs-plugin-json
**Git repository url:** https://github.com/systemjs/plugin-json
**npm package url:** https://www.npmjs.com/package/systemjs-plugin-json
**License(s):** MIT

### PR DESCRIPTION
**Pull request by pvnr0082t**
Pull request for issue: #9641 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/systemjs/plugin-json
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/systemjs-plugin-json
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 17
   - Count of watchers: 4
   - Count of forks: 21
 * NPM download stats (optional):
   - Downloads in the last day: 66
   - Downloads in the last week: 657
   - Downloads in the last month: 3035

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9641, cc @guybedford

### CODE REVIEW

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

File: ajax/libs/systemjs-plugin-json/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "systemjs-plugin-json",
+  "filename": "json.min.js",
+  "version": "0.2.2",
+  "description": "JSON loader plugin",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/systemjs/plugin-json.git"
```
Comment (x09326) [inline ajax/libs/systemjs-plugin-json/package.json]: Do we need to use `https` here?

Comment (pvnr0082t) [inline ajax/libs/systemjs-plugin-json/package.json]: @x09326 I have fixed it. Please help me review again, thank you.

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
@pvnr0082t next time if npm package is provided in request but you don't use it, please add a note about the reason.

**Comment by pvnr0082t** [comment]
@PeterDaveHello OK. I added git auto-update config due to there are 6 releases on GitHub but 4 on NPM.

**Pull request merged**